torchao/quantization/utils.py

```diff
@@ -362,8 +362,7 @@ def groupwise_affine_quantize_tensor_from_qparams(
         # Move to cpu, until issue with MPS memory management of temporary tensors is resolved
         if int_data_device_type == 'mps':
             int_data = int_data.cpu()
-        int_data = (int_data[::, ::2] << 4 | int_data[::, 1::2]).to(torch.uint8)
-        if int_data_device_type == 'mps':
+            int_data = (int_data[::, ::2] << 4 | int_data[::, 1::2]).to(torch.uint8)
```

Comment:
why is this change needed? I thought the previous version is going to handle devices other than "mps" as well

Reply:
I am not entirely sure about it as that changed was done by @jcaip. He might have a bit more context regarding this change

Reply:
I think this is just a typo on my end.